current_spring2026/ingestion/ingest.py

```
@@ -0,0 +1,394 @@
+
+    doc_ids = list({
+        ark_to_doc_id[c["ark_id"]]
+        for c in chunk_rows
+        if c["ark_id"] in ark_to_doc_id
+    })
+
+    with get_cursor(conn) as cur:
+        cur.execute("DELETE FROM chunks WHERE document_id = ANY(%s)", (doc_ids,))
+
+    rows = []
+    for chunk, emb, sparse in zip(chunk_rows, text_embeddings, sparse_embeddings):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)